Sources/SpeziLLMLocal/LLMLocalSession.swift

```diff
@@ -78,6 +78,8 @@ public final class LLMLocalSession: LLMSession, @unchecked Sendable {
     
     @MainActor public var state: LLMState = .uninitialized
     @MainActor public var context: LLMContext = []
+    /// Overrides the `context` with a custom highly customizable context in the `swift-transformers` format.
```

Comment:
As mentioned: Please update the DocC articles as well, this seems like quite a big change if we really want to go through with it. Personally, I would not introduce this property at all (as it introduces tons of side effects, duplication etc), but I'm sure you had good reasons :D

Reply:
Yes we had some issues with tool calling in the context, currently `swift-transformers` do not finalized the implementation for tool calling (see: https://github.com/huggingface/swift-transformers/blob/d42fdae473c49ea216671da8caae58e102d28709/Sources/Tokenizers/Tokenizer.swift#L357). 
However, this allows a very flexible context with different types of roles, etc.